src/orders/components/OrderPaymentSummaryCard/components/PaymentsSummary/utils.ts

```diff
@@ -0,0 +1,96 @@
+import { OrderDetailsFragment } from "@saleor/graphql";
+
+import { PaymentState } from "./types";
+
+interface ShouldDisplayResult {
+  state: PaymentState;
+  authorized: boolean;
+  captured: boolean;
+  cancelled: boolean;
+  pending: boolean;
+}
+
+export const getShouldDisplayAmounts = (
+  order: OrderDetailsFragment,
```

Comment:
question: should be order undefined or   order: OrderDetailsFragment ?

Reply:
yes, there should be `undefined` in that type, thx